### PR TITLE
Add manifest for KDE KStars

### PR DESCRIPTION
**Pull request by Pointedstick**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions.
  - `--device=dri` is for direct rendering access on Wayland
  - `--share=network` is to make the network download features work
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
  - https://invent.kde.org/ngraham
  - https://invent.kde.org/education/kstars/-/commits/master?author=Nate%20Graham
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned.
  - One patch to fix the initial color scheme has been submitted upstream at https://invent.kde.org/education/kstars/-/merge_requests/550

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: org.kde.kstars.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+    "id": "org.kde.kstars",
+    "branch": "3.5.7",
```
Comment (hfiguiere) [inline org.kde.kstars.json:3]:
```suggestion
```

File: org.kde.kstars.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kstars",
+    "rename-icon": "kstars",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
```
Comment (hfiguiere) [inline org.kde.kstars.json:11]:
```suggestion
        "--socket=fallback-x11",
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build org.kde.kstars

**Comment by hfiguiere** [comment]
```
builddir/files/share/appdata/org.kde.kstars.appdata.xml: FAILED:
• tag-missing           : <release> required
Validation of files failed
```

**Comment by Pointedstick** [comment]
bot, build org.kde.kstars

**Pull request closed without merging**